egs/multi_conv_zh_es_ta/ST/hent_srt/export.py

```
@@ -0,0 +1,640 @@
+    return parser
+
+
+class EncoderModel(nn.Module):
+    """A wrapper for encoder and encoder_embed"""
+
+    def __init__(self, model: nn.Module) -> None:
+        super().__init__()
+        self.model = model
+
+    def forward(
+        self, features: Tensor, feature_lengths: Tensor
```

Ruff (0.12.2)

Unused method argument: `features`

(ARG002)